backend/db/migrate/20220817011608_create_property_assessments.rb

```diff
@@ -2,6 +2,15 @@ class CreatePropertyAssessments < ActiveRecord::Migration[7.0]
   def change
     create_table :property_assessments do |t|
 
+      t.integer : "pid"
+      t.integer : "cm_id"
+      t.integer : "gis_id"
+      t.string : "st_num"
+      t.string : "st_name"
+      t.integer : "unit_num"
+      t.string : "city"
+      t.integer : "zipcode"
```

Comment:
```suggestion
      t.integer :pid
      t.integer :cm_id
      t.integer :gis_id
      t.string :st_num
      t.string :st_name
      t.integer :unit_num
      t.string :city
      t.integer :zipcode
```

When running this migration locally there was a Syntax error:
```
SyntaxError: /Users/mzagaja/Developer/urban-league-heat-pump-accelerator/backend/db/migrate/20220817011608_create_property_assessments.rb:5: syntax error, unexpected ':'
      t.integer : "pid"
                ^
/Users/mzagaja/Developer/urban-league-heat-pump-accelerator/backend/db/migrate/20220817011608_create_property_assessments.rb:6: syntax error, unexpected ':'
      t.integer : "cm_id"
                ^
/Users/mzagaja/Developer/urban-league-heat-pump-accelerator/backend/db/migrate/20220817011608_create_property_assessments.rb:7: syntax error, unexpected ':'
      t.integer : "gis_id"
                ^
/Users/mzagaja/Developer/urban-league-heat-pump-accelerator/backend/db/migrate/20220817011608_create_property_assessments.rb:8: syntax error, unexpected ':'
      t.string : "st_num"
               ^
/Users/mzagaja/Developer/urban-league-heat-pump-accelerator/backend/db/migrate/20220817011608_create_property_assessments.rb:9: syntax error, unexpected ':'
      t.string : "st_name"
               ^
/Users/mzagaja/Developer/urban-league-heat-pump-accelerator/backend/db/migrate/20220817011608_create_property_assessments.rb:10: syntax error, unexpected ':'
      t.integer : "unit_num"
                ^
/Users/mzagaja/Developer/urban-league-heat-pump-accelerator/backend/db/migrate/20220817011608_create_property_assessments.rb:11: syntax error, unexpected ':'
      t.string : "city"
               ^
/Users/mzagaja/Developer/urban-league-heat-pump-accelerator/backend/db/migrate/20220817011608_create_property_assessments.rb:12: syntax error, unexpected ':'
      t.integer : "zipcode"
```

The suggestion above should fix that for you!